examples/beam/src/test/java/edu/snu/nemo/examples/beam/PartitionWordsByLengthITCase.java

```diff
@@ -71,13 +69,4 @@ public void test() throws Exception {
         .addOptimizationPolicy(DefaultPolicyParallelismFive.class.getCanonicalName())
         .build());
   }
-
-  @Test (timeout = TIMEOUT)
-  public void testSailfish() throws Exception {
```

Comment:
Can you re-enable this test?

Reply:
I will re-enable the large shuffle policy tests.